python-threatexchange/threatexchange/exchanges/signal_exchange_api.py

```diff
@@ -332,6 +334,15 @@ def naive_convert_to_signal_type(
             s_type = type_by_name.get(type_str)
             if s_type is None or metadata is None:
                 continue
+            try:
+                s_type.validate_signal_str(signal_str)
+            except ValueError:
+                logging.warning(
+                    "Invalid fingerprint (%s): %s",
+                    s_type.get_name(),
+                    signal_str,
```

Comment:
blocking: This string could be quite long, suggest truncating it and doing a "..."